src/frontend/Navigation/ScreenGroups/TabBar/CameraTabBarIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import React, {FC} from 'react';
+import {TabBarIconProps, TabName} from '../../types';
+import {TabBarIcon} from './TabBarIcon';
+import {useTabNavigationStore} from '../../../hooks/useTabNavigationStore.ts';
+
+export const CameraTabBarIcon: FC<TabBarIconProps> = props => {
+  const {currentTab} = useTabNavigationStore();
+
+  return (
+    <TabBarIcon
+      {...props}
+      isFocused={props.focused && currentTab !== TabName.Tracking}
```

Comment:
If suggestion in TabBarIcon.tsx is applied:

```suggestion
      focused={props.focused && currentTab !== TabName.Tracking}
```